app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from __future__ import annotations
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    NotWearingMaskError,
+    OutdatedVaccineError
+)
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str | None:
+    mask_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except NotWearingMaskError:
+            mask_to_buy += 1
+        except (NotVaccinatedError, OutdatedVaccineError):
+            return "All friends should be vaccinated"
```

Comment:
"Create a VaccineError parent class for NotVaccinatedError and OutdatedVaccineError errors. Use it to catch both types of errors in the same except clause. " Use this tip here